### PR TITLE
Test status checks

### PR DESCRIPTION
**Pull request by patrickersing**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -51,7 +51,6 @@ jobs:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
       - run: julia -e 'using InteractiveUtils; versioninfo(verbose=true)'
-      - uses: julia-actions/cache@v1
```
Comment (ranocha) [inline .github/workflows/ci.yml:54]: ?

**Comment by sloede** [comment]
I've re-enabled the CI checks to be required, but not (yet) the coveralls check.

**Comment by sloede** [comment]
So, I made coveralls non-required again and instead made codecov required. This seems to work as expected:
image
Maybe we keep it like this for now and move on to other topics 😅

**Comment by patrickersing** [comment]
Nice! This seems to fix our problem, for me it's totally fine to keep it like this.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (patrickersing) [reply .github/workflows/ci.yml:54]: But there seem to be some problems with deleting these caches, so should we give it a try and set the permissions?

Comment (sloede) [reply .github/workflows/ci.yml:54]: Sure, give it a try!

**Comment by andrewwinters5000** [comment]
> Maybe we keep it like this for now and move on to other topics 😅

That is okay from my end of things as well. Thanks for your help @sloede !

**Pull request closed without merging**